python/lsst/ts/Dome/mock_llc_statuses/lwscs_status.py

```diff
@@ -102,7 +100,7 @@ async def moveEl(self, elevation):
             self.motion_direction = motion_dir.DOWN.value
 
     async def crawlEl(self, direction, velocity):
-        """Mock crawling of the light and wind screen in the given direction at the given velocity.
+        """Crawl the light and wind screen in the given direction at the given velocity.
```

Comment:
I'm curious: are we going to use this command? I thought you or German said no. If you are *sure* we won't want it then please consider taking it out.

Reply:
It is not sure yet so I will leave it for now. As soon as it is clear that we will not need it, I will remove it. It then would require a change in ts_xml as well.